search/src/kernels.cu

```diff
@@ -445,7 +454,7 @@ __global__ void searchFilterImages(int trajectoryCount, int width, int height,
         for (int r = 0; r < RESULTS_PER_PIXEL; ++r)
         {
             if (currentT.lh > best[r].lh &&
-                currentT.obsCount >= minObservations)
+                currentT.lh >= minLH)
```

Comment:
Maybe my logic is flawed, but is it possible for `currentT.lh < minLH` given the second argument in the conditional on line 377?